modules/web-console/web-console-common/src/main/java/org/apache/ignite/console/websocket/WebSocketEvents.java

```diff
@@ -58,4 +58,7 @@
 
     /** */
     public static final String CLUSTER_TOPOLOGY = "cluster:topology";
+
+    /** */
+    public static final String CLUSTER_LOGOUT = "cluster:logout";
```

Comment:
I think, after this change we need to increase web agent protocol version, should we?